src/renderer/helpers/api/local.js

```diff
@@ -787,6 +790,32 @@ export function parseLocalListPlaylist(playlist, channelId = undefined, channelN
       playlistId: lockupView.content_id,
       videoCount: extractNumberFromString(thumbnailOverlayBadgeView.badges[0].text)
     }
+  } else if (playlist.type === 'CompactStation') {
+    /** @type {import('youtubei.js').YTNodes.CompactStation} */
+    const compactStation = playlist
+
+    return {
+      type: 'playlist',
+      dataSource: 'local',
+      title: compactStation.title.text,
+      thumbnail: compactStation.thumbnail.at(1).url,
```

Comment:
Any reason you chose to use the `.at()` function with a positive number instead of using array indexing `[1]`?

Reply:
Just a personal preference, I'll switch it to [1]